style/style.css

```
@@ -0,0 +1,120 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #e5e5e5;
+}
+
+.main_color {
+  color: #6070ff;
+}
+
+.h1Color {
+  color: #091e42;
+}
+
+.pColor {
+  color: #344563;
+}
+
+.white {
+  color: #fff;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 70px;
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 30px 18%;
+  background-color: #fff;
+}
+
+.logo {
+  text-decoration: none;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.headline-icons {
+  margin: 5px;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.icon {
+  margin: 5px;
+  cursor: pointer;
+}
+
+@media only screen and (min-width: 375px) {
+  .headline {
+    margin-top: 70px;
+    height: 100vh;
+    background-image: url(../Images/headerShapesMobile.png), url(../Images/headerBackgroundMobile@2x.png);
+    background-size: cover;
+  }

```

- [ ] Kindly make remove the `height: 100vh` because it is causing the headline section to take 100% of the visible screen which is not the same as the Figma design.

```suggestion
@media only screen and (min-width: 375px) {
.headline {
    margin-top: 70px;
    padding-bottom: 4rem;
    border-bottom-left-radius: 5rem;
    background-image: url(../Images/headerShapesMobile.png), url(../Images/headerBackgroundMobile@2x.png);
    background-size: cover;
  }

```